Helper/Order.php

```diff
@@ -878,6 +878,7 @@ protected function deleteOrder($order)
         try {
             $order->cancel()->save()->delete();
         } catch (\Exception $e) {
+            $this->bugsnag->notifyException($e);
```

Comment:
@danac-gs   Do we need here some additional ex. order id or order increment_id?
As you think?

I thinking about situation when we will have an exception and additionally we already will have some additional info.

Reply:
I think the exception message will have most of that information, but I could be mistaken.

Reply:
@YevhenShBolt IDs are added to log. Please approve.